src/main/java/org/mqttbee/api/mqtt/MqttClientBuilder.java

```diff
@@ -59,17 +67,48 @@ public MqttClientBuilder forServerHost(@NotNull final String host) {
     }
 
     @NotNull
-    public MqttClientBuilder forServerPort(final int port) {
+    public MqttClientBuilder withServerPort(final int port) {
         this.serverPort = port;
         return this;
     }
 
+    @NotNull
+    public MqttClientBuilder withServerPath(final String serverPath) {
```

Comment:
If this only applies when using websockets, then add a own config
e.g. instead of withWebsockets and withServerPath: withWebsocketConfig(MqttWebsocketConfig)

Reply:
done